modules/admin_manual/pages/installation/ubuntu_18_04.adoc

```diff
@@ -168,20 +169,30 @@ If you need to sync your users from an LDAP or Active Directory Server, add this
 
 [source,console,subs="attributes+"]
 ----
-echo "*  */6  *  *  * /usr/bin/php -f {install-directory}/cron.php" \
-  > /var/spool/cron/crontabs/{webserver-user}
-sudo -u {webserver-user} chown {webserver-user}.crontab /var/spool/cron/crontabs/{webserver-user}
-sudo -u {webserver-user} chmod 0600 /var/spool/cron/crontabs/{webserver-user}
+echo "*/15  *  *  *  * /usr/bin/php -f /var/www/owncloud/cron.php" > /var/spool/cron/crontabs/www-data
```

Comment:
Maybe use /etc/cron.d instead of manipulating the crontab?

Reply:
this was done so no user interaction is required.

If you have a command line command that we can use instead - feel free to share :)

Reply:
The folder /var/spool/cron is not meant to be manipulated like that. For this use case it's common to use /etc/cron.d

Reply:
We are emulating somebody editing the file as www-data using crontab -e so they can use further documentation using this command.